test/DotNetWorkerTests.Shared/TestFunctionBindingsFeature.cs

```diff
@@ -6,7 +6,7 @@
 using Microsoft.Azure.Functions.Worker.Context.Features;
 using Microsoft.Azure.Functions.Worker.OutputBindings;
 
-namespace DotNetWorker.OpenTelemetry.Tests
+namespace Microsoft.Azure.Functions.Worker.Tests.Shared
```

Comment:
Consider dropping `.Shared` from the namespace here and throughout this PR. I personally like putting these common types in a more common namespace for other test projects to easily use.